### ISSUE TITLE
Improve form accessibility

### ISSUE THREAD

**Issue by ElliotThiebaut** [medrunner-services/Client-Portal#71]
There is a lot of buttons and input submissions across the app but not all are wrapped into an HTML `<form>` element. This poses issues for screen readers and keyboard input failover, so mostly accessibility.

Input submissions in this context is defined by an association of an input that requires user interaction and a button that requires a second interaction to submit the action.

The goal is to :

- Ensure all input submissions that should are wrapped in a form element if possible.
- Add the ability to use only the keyboard threw the form with tab to navigate threw the inputs and enter to submit the form.
- Add the ability to confirm with a tabulation and an enter press a confirmation modal.

**Comment by rabelmervin** [comment]
Hi sir @ElliotThiebaut, I'm interested to work on this issue sir, Could you please tell me the location of the code ???

**Comment by ElliotThiebaut** [comment]
> Hi sir @ElliotThiebaut, I'm interested to work on this issue sir, Could you please tell me the location of the code ???

@rabelmervin As the issue description said, the goal is to go look across the whole codebase to check, with a Vue app like this, everything in the src directory!
